great_expectations/data_context/data_context.py

```diff
@@ -3264,6 +3264,38 @@ def run_checkpoint(
             **kwargs,
         )
 
+    def add_profiler(
+        self,
+        name: str,
+        config_version: float,
+        rules: Dict[str, dict],
```

Comment:
can this be Dict[str, Rules]? Where do the values that would be passed to this method come from?

Reply:
I don't believe so - the schema requires primitive types (see `RuleBasedProfilerConfig`'s constructor).

This method will either take the result of parsed YAML or be invoked by the user in their own script. I think either way, it's a dictionary at this point in the program.